source/_integrations/hive.markdown

```diff
@@ -17,11 +17,22 @@ ha_codeowners:
 ha_domain: hive
 ---
 
-The `hive` integration is the main integration to set up and integrate all supported Hive devices. Once configured with the minimum required details it will detect and add all Hive devices into Home Assistant, including support for multi-zone heating.
+This `hive` integration uses the same username and password you use on the [Hive website](https://my.hivehome.com) to configure it within Home Assistant. Once configured Home Assisatnt will detect and add all Hive devices, including support for multi-zone heating.
```

Comment:
It is generally wise to start with an introduction. If one gets to this page, the first thing they read about this integration is something about a Hive username/password. But what is Hive?

```suggestion
This Hive integration uses the same username and password you use on the [Hive website](https://my.hivehome.com) to configure it within Home Assistant. Once configured Home Assistant will detect and add all Hive devices, including support for multi-zone heating.
```